postgres_copy/copy_to.py

```diff
@@ -24,10 +24,11 @@ def setup_query(self):
             for field in self.query.copy_to_fields:
                 # raises error if field is not available
                 expression = self.query.resolve_ref(field)
-                if field in self.query.annotations:
```

Comment:
This change seems like a good one, but unrelated to insert_count. Is that right, or am I missing something?